Common/Utils/include/CommonUtils/BoostHistogramUtils.h

```diff
@@ -215,24 +219,24 @@ std::string createErrorMessage(o2::utils::FitGausError_t errorcode);
 /// \param first begin iterator of the histogram
 /// \param last end iterator of the histogram
 /// \param axisFirst axis iterator over the bin centers
-/// \param ignoreUnderOerflowBin switch to disable taking under and overflow bin into fit
+/// \param ignoreUnderOverflowBin switch to disable taking under and overflow bin into fit
 /// \return result
 ///      result is of the type fitResult, which contains 4 parameters (0-Constant, 1-Mean, 2-Sigma,  3-Sum)
 ///
 /// ** Temp Note: For now we forgo the templated struct in favor of a std::vector in order to
 /// have this compile while we are working out the details
 template <typename T, typename Iterator, typename BinCenterView>
-std::vector<double> fitGaus(Iterator first, Iterator last, BinCenterView axisfirst, const bool ignoreUnderOerflowBin = true)
+std::vector<double> fitGaus(Iterator first, Iterator last, BinCenterView axisfirst, const bool ignoreUnderOverflowBin = true)
 {
 
-  if (ignoreUnderOerflowBin) {
+  if (ignoreUnderOverflowBin) {
     first++;
     last--;
   }
 
-  TLinearFitter fitter(3, "pol2");
-  TMatrixD mat(3, 3);
-  Double_t kTol = mat.GetTol();
+  static TLinearFitter fitter(3, "pol2");
+  static TMatrixD mat(3, 3);
+  static Double_t kTol = mat.GetTol();
```

Comment:
I suspect this is to circumvent the problem with the multiple definition of TFormula when running with OMP threads > 1. This could become tricky in a multi-threaded environment. As far as I understand the variables should be allocated per thread (thread-local storage) based on our build settings. However would be good if we could find a way to avoid making members static.